composer.json

```diff
@@ -24,7 +24,8 @@
             "dev-master": "3.0-dev"
         }
     },
+    "minimum-stability": "dev",
     "require-dev": {
-        "phpunit/phpunit": "^6.5"
+        "phpunit/phpunit": "9.3.x-dev"
```

Comment:
An alternative is to use Symfony PHPUnit Bridge, which is especially designed to handle multiple PHP versions and multiple PHPUnit versions. It's what we do for API Platform for instance.